### PR TITLE
impl isPrimitive

### PR DESCRIPTION
**Pull request by BnjmnZmmrmn**
**contents**
- author names, game name, and db name metadata
- generic_hash initialization
- Primitive function implementation

**reviewers**
- we loop through indexes 0-7 because we only need to look at half the board
- for each idx we check if we can check left and right for wins. each idx can also check down, so we check diagonals too
- if we find a 3 in a row, we save the idx and break
- outside our loop, if we found 3 in a row, we return win/loss conditional to character at the idx wrt player

### CODE REVIEW

File: src/mjan.c [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
 **
 ************************************************************************/
 
-#include "gamesman.h"
+#include "gamesman.h" 
 
-CONST_STRING kAuthorName = "Firstname Lastname";
-CONST_STRING kGameName = "Your Game Name";  // Use this spacing and case
-CONST_STRING kDBName = "yourgamename";      // Use this spacing and case
+CONST_STRING kAuthorName = "Andy Kim Aj DeMarinas Benjamin Zimmerman";
```
Comment (ajdema) [inline src/mjan.c]: DeMarinis but idc

Comment (BnjmnZmmrmn) [reply src/mjan.c]: sg done

File: src/mjan.c [hunk 2]
```diff
@@ -185,6 +186,41 @@ POSITION DoMove(POSITION position, MOVE move) {
  * src/core/types.h for the value enum definition.
  */
 VALUE Primitive(POSITION position) {
+    char* p_str = (char*)SafeMalloc(possize);
+    p_str = generic_hash_unhash(position, p_str);
+    int deciding_idx = -1;
+    for (int idx = 0; idx < 8; idx ++) {
+        if (p_str[idx] == '-') {
+            break;
```
Comment (ajdema) [inline src/mjan.c]: continue here

**Review by ajdema — APPROVED** [review]

File: src/mjan.c (hunk 2 above)
Comment (BnjmnZmmrmn) [reply src/mjan.c]: sg done

**Pull request merged**